terminus-ui/table/src/cell.ts

```diff
@@ -237,18 +213,25 @@ export class TsHeaderCellDirective extends CdkHeaderCell implements AfterViewIni
     resizeElement.title = 'Drag to resize column.';
     this.renderer.appendChild(this.elementRef.nativeElement, resizeElement);
 
-    // Remove any existing event listeners
-    this.removeAllEventListeners();
-    this.resizeListenerRemover = this.renderer.listen(resizeElement, 'mousedown', e => this.onResizeColumn(e));
+    // Handle column resize events
+    fromEvent<MouseEvent>(resizeElement, 'mousedown')
+      .pipe(untilComponentDestroyed<MouseEvent>(this))
+      .subscribe(e => this.onResizeColumn(e));
```

Comment:
like how clean this is 👍

Reply:
Yeah, this wasn't my first pass at it 😆 but really dig using RxJS for DOM events. My first pass was at least double the line count